src/util/getReceiverAddress.ts

```diff
@@ -46,15 +45,14 @@ const getReceiverAddress = ({ targetChainDetails, sourceChainDetails, receiverAd
   };
 
   try {
-    const [validatedDerivedAcccount, rest] = checkAddress(receiverAddress, targetSS58Format);
+    const decodedReceiverAddress = base58Decode(receiverAddress);
+    const [isValidDerivedAcccount, , , ss58Decoded] = checkAddressChecksum(decodedReceiverAddress);
```

Comment:
:sob: my eyes hurt when I look at the `, , , ` to skip params, but I guess it's a common JS thing, isn't it?

Could we maybe add something like `isValidDerivedAccount, _length, _ss58Lengh, ss58Decoded` for readabiliy, i.e. some unused variables without making `eslint` yell at us?

Reply:
Totally agree on these unused vars :( 

The only way to keep ESLint happy here is to at a `//eslint-disable ...` IF we want to keep the variables named but unused, otherwise we can modify the `.eslintrc` to support unused variables but...that also sucks 👎🏻